contracts/Deposit.sol

```diff
@@ -0,0 +1,84 @@
+pragma solidity ^0.4.19;
+
+contract AssetManager{
+    /*
+    * Standard interface contract for deploying SimpleDeposit contracts
+    */
+    
+    address public creator;
+    address[] public deposits;
+    uint public minimum_deposit_threshold = 0.2 ether;
+    uint public minimum_fee = 0.1 ether;
+    
+    function AssetManager() public{
+        creator = msg.sender;
+    }
+    
+    function createDeposit(address assetOwner, uint fee, uint daysUntilExpiry) payable public{
```

Comment:
in the scaled back implementation, **fee** should a transaction fee paid to cinch upon initialization of the deposit.  It's not passed by the depositmaker.  It can be hardcoded within the smart contract. 
the asset rental fee is beyond the scope of the first version.